src/searchengine/nova/socks.py

```diff
@@ -200,7 +200,7 @@ def __negotiatesocks5(self,destaddr,destport):
             if authstat[1] != "\x00":
                 # Authentication failed
                 self.close()
-                raise Socks5AuthError,((3,_socks5autherrors[3]))
+                raise Socks5AuthError((3,_socks5autherrors[3]))
```

Comment:
Third party code, unless it is broken we don't change it (applies to whole file).